contrib/checkbox-ce-oem/checkbox-ce-oem-snap/series_classic20/snap/snapcraft.yaml

```
@@ -66,13 +65,14 @@ parts:
       - python3-requests-oauthlib
   input-pcspkr:
     plugin: nil
+    after: [checkbox-provider-ce-oem]
     build-packages:
       - gcc
       - libc6-dev
     override-build: |
       mkdir -p ${SNAPCRAFT_PART_INSTALL}/usr/bin
-      gcc ${SNAPCRAFT_PROJECT_DIR}/parts/checkbox-provider-ce-oem/src/checkbox-provider-ce-oem/src/input-pcspkr.c -o ${SNAPCRAFT_PROJECT_DIR}/parts/checkbox-provider-ce-oem/src/checkbox-provider-ce-oem/src/beep
-      cp -v ${SNAPCRAFT_PROJECT_DIR}/parts/checkbox-provider-ce-oem/src/checkbox-provider-ce-oem/src/beep ${SNAPCRAFT_PART_INSTALL}/usr/bin/
+      gcc ${SNAPCRAFT_PROJECT_DIR}/parts/checkbox-provider-ce-oem/src/src/input-pcspkr.c -o ${SNAPCRAFT_PROJECT_DIR}/parts/checkbox-provider-ce-oem/src/src/beep
+      cp -v ${SNAPCRAFT_PROJECT_DIR}/parts/checkbox-provider-ce-oem/src/src/beep ${SNAPCRAFT_PART_INSTALL}/usr/bin/
```

```suggestion
```suggestion
      PART_SRC="${SNAPCRAFT_PROJECT_DIR}"/parts/checkbox-provider-ce-oem/src
      gcc "$PART_SRC"/src/input-pcspkr.c -o $snapcraft_parts_src/src/beep
      cp -v "$PART_SRC"/src/beep ${SNAPCRAFT_PART_INSTALL}/usr/bin/
```
```